.github/workflows/release-winget.yaml

```diff
@@ -7,26 +7,30 @@ jobs:
   release:
     runs-on: ubuntu-latest
     steps:
-    - name: Update winget repository
-      uses: mjcheetham/update-winget@v1.0
+    - id: update-winget
+      name: Update winget repository
+      uses: mjcheetham/update-winget@v1.2
       with:
-        token: ${{ secrets.WINGET_TOKEN }}
-        repo: microsoft/winget-pkgs
         id: Microsoft.GitCredentialManagerCore
+        token: ${{ secrets.WINGET_TOKEN }}   
         releaseAsset: gcmcore-win-x86-(.*)\.exe
+        repo: ldennington/winget-playground
```

Comment:
We'll want to change this before merging.

Reply:
Phew! Good catch.